cmd/server-main.go

```diff
@@ -609,7 +610,7 @@ func serverMain(ctx *cli.Context) {
 	}
 
 	// Prints the formatted startup message, if err is not nil then it prints additional information as well.
-	printStartupMessage(getAPIEndpoints(), err)
+	go printStartupMessage(getAPIEndpoints(), err)
```

Comment:
This doesn't seem very important besides it can overlap with logs that is coming later

Reply:
It is important since it calls StorageInfo() API call